etc/otel-jmx.config.yaml

```diff
@@ -12,25 +11,46 @@ rules:
        metric: kafka.streams.records.lag
        type: gauge
        desc: the current lag of the partition
-       unit: '{messages}'
+       unit: '{records}'
```

Comment:
I'm going to change to "offsets" - make sense @ableegoldman

Reply:
yeah that's fine, was just thinking we should standardize units but I don't think it really matters too much